src/Map/MapComponent/MapComponent.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OlMap from 'ol/map';
+
+/**
+ * Class representating a map.
+ *
+ * @class The Map.
+ * @extends React.Component
+ */
+export class MapComponent extends React.Component {
+
+  /**
+   * The properties.
+   * @type {Object}
+   */
+  static propTypes = {
+    map: PropTypes.instanceOf(OlMap).isRequired,
+    mapDivId: PropTypes.string
+  }
+
+  /**
+   * The default properties.
+   * @type {Object}
+   */
+  static defaultProps = {
+    mapDivId: 'map'
+  }
+
+  /**
+   * Create a MapComponent.
+   *
+   * @constructs Map
+   */
+  constructor(props) {
+    super(props);
+  }
+
+  /**
+   * The componentDidMount function
+   *
+   * @method componentDidMount
+   */
+  componentDidMount() {
+    this.props.map.setTarget('map');
+  }
+
+  /**
+   * The render function.
+   */
+  render() {
+    let mapDiv;
+
+    const {
+      map,
+      mapDivId,
+      ...passThroughProps
+    } = this.props;
+
+    if (map) {
+      mapDiv = <div
+        className="map"
+        id={mapDivId}
+        {...passThroughProps}
```

Comment:
Please add the possibility to pass `children` to the `div` element.

Reply:
Solved.